TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
+  
+  private let markedSpaces: [Int: PlayerMark]
+  private let size = 3
+
+  public init() {
+    markedSpaces = [:]
+  }
+
+  public init(markedSpaces: [Int : PlayerMark]) {
```

Comment:
Nope. As a fix, I've implemented a helper method that accomplishes the same thing in the test file instead. Let me know if you think there's a better solution.